go/client/files.go

```diff
@@ -80,6 +82,14 @@ func (b *StdinSource) Close() error {
 	return err
 }
 
+func (b *StdinSource) CloseWithError(e error) error {
```

Comment:
If there was an error (say if the identify2 portion of encrypt failed), then no need to consume all input. We can just crash out.